### PR TITLE
#170947496 edit and view profile

### PR DESCRIPTION
**Pull request by izzett222**
#### What does this PR do?
this PR adds the profile page where use can view and edit their profile information
#### Description of Task to be completed?
- add the navbar for an authenticated user
- add profile component
- add tests for profile component
- add the sidebar to navigate to different pages
#### How should this be manually tested?
- clone this repo
- run npm install
- run npm run test(for tests in development)
- run npm start:dev
- login through url/login
- switch to the url/profile on the same window
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947496](https://www.pivotaltracker.com/n/projects/2431451/stories/170947496)
#### Questions:

### CODE REVIEW

**Review by bbaime98 — APPROVED** [review]

**Review by Ntare22 — APPROVED** [review]

**Review by Baraka-Mugisha — APPROVED** [review]

File: src/components/sharedComponents/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    return (
+      <div className="wrapper">
+        <div className="sidebar" style={style}>
+          <Route>
+            {this.props.children}
+          </Route>
+        </div>
+        <button type="button" style={buttonStyle} onClick={this.handleChange}>{slide ? 'close' : 'open'}</button>
+      </div>
+    );
```
Comment (Baraka-Mugisha) [inline src/components/sharedComponents/Sidebar.js]: You could add a translation for these (`close` and `open`) too!

**Comment by Baraka-Mugisha** [comment]
The page is not scrollable on my end. I can not access the `save` button.
scrollable

File: src/components/sharedComponents/Sidebar.js (hunk 1 above)
Comment (izzett222) [reply src/components/sharedComponents/Sidebar.js]: I added the translation, check out the Heroku app to test out the changes

**Comment by jabichris** [comment]
Screen Shot 2020-05-18 at 19 48 57
Nice work @izzett222 . Can you please add an image placeholder on the profile picture? take time also to style the div that contains profile settings so that it looks nice.

**Pull request merged**